demo/demo-plugins/footer.js

```diff
@@ -0,0 +1,28 @@
+//  Append footer with the Vercel logo to every page.
+// TODO: Check if there a way to server-side render this someday?
```

Comment:
It would be much better to run all of this once on the server side at compile time, but the execution cost is negligible so I'll leave it as is for now. Note that the resulting HTML string is constant every time, so we could technically preval `sponsorContainer` once, and use it the same string everywhere.